.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Please do not commit your .vscode folder - this can be added to the .gitignore

Reply:
hi can you please help me how i can change it now